libs/eslint-plugin/project.json

```diff
@@ -14,12 +13,12 @@
       "executor": "@nrwl/jest:jest",
       "outputs": ["coverage/libs/eslint-plugin"],
       "options": {
-        "jestConfig": "libs/eslint-plugin/jest.config.js",
+        "jestConfig": "libs/eslint-plugin/jest.config.ts",
         "passWithNoTests": true
       }
     },
     "build": {
-      "executor": "@nrwl/node:package",
+      "executor": "@nrwl/js:tsc",
```

Comment:
question: Is this executor recommended for Node.js packages?

Reply:
It looks like yes, this is the default executor after generating the`@nrwl/node:library`